kani-compiler/src/codegen_cprover_gotoc/compiler_interface.rs

```diff
@@ -258,3 +261,23 @@ where
         serde_json::to_writer(writer, &source).unwrap();
     }
 }
+
+/// Prints a report at the end of the compilation.
+fn print_report<'tcx>(ctx: &GotocCtx, tcx: TyCtxt<'tcx>) {
+    // Print all unsupported constructs.
+    if !ctx.unsupported_constructs.is_empty() {
+        // Sort alphabetically.
+        let unsupported: BTreeMap<String, &Vec<Location>> = ctx
+            .unsupported_constructs
+            .iter()
+            .map(|(key, val)| (key.map(|s| String::from(s)), val))
+            .collect();
+        let mut msg =
+            String::from("Compilation unit included the following unsupported constructs:\n");
```

Comment:
Is "Compilation unit" familiar terminology to our users?

Reply:
I changed the message. Let me know what you think.